slatedb/src/mem_table_flush.rs

```diff
@@ -155,6 +155,12 @@ impl MemtableFlusher {
                     modifier.state.manifest.core.recent_snapshot_min_seq =
                         min_active_snapshot_seq.unwrap_or(modifier.state.manifest.core.last_l0_seq);
 
+                    // update all the sequence numbers in the manifest
+                    let seq_tracker = self.db_inner.oracle.seq_tracker.read().map_err(|e| {
```

Comment:
we should write this out when writing the manifest as part of a clean shutdown (when the memtable flush task exits)